--- node_modules/libsql/index.js.orig
+++ node_modules/libsql/index.js
@@ -6,6 +6,10 @@
 function requireNative() {
   if (process.env.LIBSQL_JS_DEV) {
     return load(__dirname)
+  }
+  // 在 Mac ARM64 环境中强制使用 darwin-arm64
+  if (process.platform === 'darwin' && process.arch === 'arm64') {
+    return require('@libsql/darwin-arm64');
   }
   let target = currentTarget();
   // Workaround for Bun, which reports a musl target, but really wants glibc... 